client/lib/post-normalizer/rule-content-make-images-safe.js

```diff
@@ -60,7 +60,7 @@ function isCandidateForContentImage( imageUrl ) {
 	} );
 }
 
-export default function( maxWidth ) {
```

Comment:
we're not, but i'm not sure no one is. If we drop it, we also have to update callers too to not invoke the function to get the actual normalization function.